app/views/claims/support/support_users/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for(:page_title) { t(".page_title") } %>
```

Comment:
Simon pointed out to me yesterday that if there are errors on the page, the page title should be prefixed with "Errors: " It looks like the prototypes are busted, so I can't screen shot an example, but I did it correctly for adding provider / school users as non-support users on the placements side. Something like
`<%= content_for :page_title, @support_user.errors.any? ? t(".page_title_with_error") : t(".page_title") %>`